dev-support/database/init-database.sh

```diff
@@ -39,7 +39,6 @@ mysql -e "CREATE USER IF NOT EXISTS 'metastore'@'%' IDENTIFIED BY 'password';"
 mysql -e "GRANT ALL PRIVILEGES ON *.* TO 'metastore'@'%';"
 mysql -e "use metastore; source ./metastore.sql; show tables;"
 
-mysql -e "CREATE DATABASE IF NOT EXISTS mlflow;"
```

Comment:
Why mlflow is renamed to mlflowdb?

Reply:
I don't know the reason why it fails when I name the database as mlflow. But when I name the database as mlflowdb, it works.